### ISSUE TITLE
Breaking Juke Box while song is playing causes game to crash

### ISSUE THREAD

**Issue by salem-5** [tomalbrc/VisualJukebox#2]
```
[17:42:02] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Exception ticking world
        at knot/net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1073) ~[server-intermediary.jar:?]
        at knot/net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:953) ~[server-intermediary.jar:?]
        at knot/net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:713) ~[server-intermediary.jar:?]
        at knot/net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:292) ~[server-intermediary.jar:?]
        at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "eu.pb4.polymer.virtualentity.api.elements.ItemDisplayElement.setInterpolationDuration(int)" because "this.discElement" is null
        at knot/de.tomalbrc.visualjukebox.JukeboxHolder.tick(JukeboxHolder.java:76) ~[visualjukebox-1.0.0+1.21.2-rc1.jar:?]
        at knot/eu.pb4.polymer.virtualentity.api.attachment.ChunkAttachment.tick(ChunkAttachment.java:87) ~[polymer-virtual-entity-0.10.0+1.21.2-a1d943057abc818f.jar:?]
        at knot/net.minecraft.class_3215.handler$bad000$polymer-virtual-entity$tickElementHoldersEvenIfBlocksDont(class_3215.java:1143) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_3215.method_61265(class_3215.java:426) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_3215.method_14161(class_3215.java:356) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_3215.method_12127(class_3215.java:326) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_3218.method_18765(class_3218.java:354) ~[server-intermediary.jar:?]
        at knot/net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1069) ~[server-intermediary.jar:?]
        ... 4 more
        
```

here is the crash report:
steps to reproduce:
put music disc in jukebox and break it in creative

i had static option to false in the config when this occured

**Comment by tomalbrc** [comment]
Thanks for the report ^^
fixed in 1.0.1

**Comment by Keaton42** [comment]
Is there a 1.21.1 version where you have the 1.0.1? I want to use the mod, thanks!
